akka-http-core/src/main/java/akka/http/javadsl/model/Uri.java

```diff
@@ -171,6 +171,12 @@
    */
   public abstract String getPathString();
 
+  /**
+   * Returns the scaladsl view of this instance
```

Comment:
same as bellow, explain that it mirrors the APIs, no impl details in such methods pls